src/main/java/com/stripe/model/Person.java

```diff
@@ -0,0 +1,109 @@
+package com.stripe.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import com.stripe.exception.StripeException;
+import com.stripe.net.ApiResource;
+import com.stripe.net.RequestOptions;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+
+import lombok.AccessLevel;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = false)
+public class Person extends ApiResource implements MetadataStore<Person>, HasId {
+  @Getter(onMethod = @__({@Override})) String id;
+  String object;
+  String account;
+  Address address;
+  JapanAddress addressKana;
+  JapanAddress addressKanji;
+  Long created;
+  Boolean deleted;
+  DateOfBirth dob;
+  String email;
+  String firstName;
+  String firstNameKana;
+  String firstNameKanji;
+  String gender;
+  Boolean idNumberProvided;
+  String lastName;
+  String lastNameKana;
+  String lastNameKanji;
+  String maidenName;
+  @Getter(onMethod = @__({@Override})) Map<String, String> metadata;
+  String phone;
+  Requirements requirements;
+  @SerializedName("ssn_last_4_provided") Boolean ssnLast4Provided;
+  Verification verification;
+
+  // <editor-fold desc="delete">
+  /**
+   * Delete a person.
+   */
+  public Person delete() throws StripeException {
+    return delete((RequestOptions) null);
+  }
+
+  /**
+   * Delete a person.
+   */
+  public Person delete(RequestOptions options) throws StripeException {
+    return request(RequestMethod.DELETE, this.getInstanceUrl(), null, Person.class, options);
+  }
+  // </editor-fold>
+
+  // <editor-fold desc="update">
+  /**
+   * Update a person.
+   */
+  public Person update(Map<String, Object> params) throws StripeException {
+    return update(params, (RequestOptions) null);
+  }
+
+  /**
+   * Update a person.
+   */
+  public Person update(Map<String, Object> params, RequestOptions options)
+      throws StripeException {
+    return request(RequestMethod.POST, this.getInstanceUrl(), params, Person.class, options);
+  }
+  // </editor-fold>
+
+  protected String getInstanceUrl() {
+    if (this.account != null) {
+      return String.format("%s/%s/persons/%s", classUrl(Account.class), this.getAccount(),
+          this.getId());
+    }
+    return null;
+  }
+
+  @Getter
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public static class Relationship extends StripeObject {
+    Boolean accountOpener;
+    Boolean director;
+    Boolean executive;
+    Boolean owner;
+    BigDecimal percentOwnership;
+    Boolean representative;
+    String title;
+  }
+
+  @Getter
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public static class Requirements extends StripeObject {
```

Comment:
You mentioned that this class might also be used directly on accounts in the future. Maybe we should not make it an inner class then, to avoid a breaking change later.

Reply:
Ack. I feel like it was a real mistake to make those inner classes if moving them later is a breaking change. I think we should actively consider undoing that so that classes can just be re-used if they end up being shared.

Reply:
Yeah. We briefly discussed it in #547. @brandur-stripe said:
> There might be some complications around subresources that are shared like you mentioned with `Address` — occasionally these might appear as linked to only one class originally, but then be introduced in another class, so we'd have to promote them to the top level. I don't think it happens often enough to be a major problem though.

We might have been overly optimistic here :/